src/extension/app.tsx

```diff
@@ -41,9 +41,11 @@ chrome.runtime.sendMessage({ fetch: true }, function (response) {
             size += entry.length;
             let json = JSON.parse(uncompress(entry.payload));
             let tabId = entry.tabId;
+            let dateTime = entry.dateTime;
```

Comment:
>let dateTime = entry.dateTime; [](start = 12, length = 30)

nit: this seems a little more roundabount than doing 
json.envelope.dateTime = entry.dateTime then getting rid of line 48